examples/multi-account-multi-region-auto-snapshot/management_account.tf

```diff
@@ -0,0 +1,51 @@
+provider "aws" {
+  // Set this profile to the AWS Organizations management account profile.
+  // profile = "management-account"
+  alias  = "management-account-usw1"
+  region = "us-west-1"
+}
+
+// Create the required role for the management account.
+module "lacework_aws_agentless_management_scanning_role" {
+  source = "../.."
+
+  providers = {
+    aws = aws.management-account-usw1
+  }
+
+  snapshot_role           = true
+  global_module_reference = module.lacework_aws_agentless_scanning_global
+}
+
+resource "aws_cloudformation_stack_set" "snapshot_role" {
+  provider = aws.management-account-usw1
+
+  auto_deployment {
+    enabled                          = true
+    retain_stacks_on_account_removal = false
+  }
+
+  capabilities     = ["CAPABILITY_NAMED_IAM"]
+  description      = "Lacework AWS Agentless Workload Scanning Organization Roles"
+  name             = "lacework-agentless-scanning-stackset"
+  permission_model = "SERVICE_MANAGED"
+
+  parameters = {
+    ExternalId         = module.lacework_aws_agentless_scanning_global.external_id
+    ECSTaskRoleArn     = module.lacework_aws_agentless_scanning_global.agentless_scan_ecs_task_role_arn
+    ResourceNamePrefix = module.lacework_aws_agentless_scanning_global.prefix
+    ResourceNameSuffix = module.lacework_aws_agentless_scanning_global.suffix
+  }
+
+  template_url = "https://agentless-workload-scanner.s3.amazonaws.com/cloudformation-lacework/latest/snapshot-role.json"
```

Comment:
Curious to know how do we deliver this template?

Reply:
The template is publicly accessible - it primarily gets used as a sub-template for the CloudFormation deployments that we run for AWLS.  If you look at how they deploy, there are three templates - global scan resources, regional scan resources, and then the cross-account role that the scanners use.

Reply:
To further add on, this template is used by the generated CloudFormation template if you download it from the platform UI (or click the direct run button). So this template is not new for what Allan is adding. The design, development, and release of this template is already mature and streamlined.